includes/Modules/Sign_In_With_Google.php

```diff
@@ -713,7 +714,7 @@ protected function inline_module_data( $modules_data ) {
 			$inline_data['existingClientID'] = $existing_client_id;
 		}
 
-		$is_woocommerce_active            = $this->woocommerce->is_active();
+		$is_woocommerce_active            = $this->is_woocommerce_active;
```

Comment:
This redeclaration seems needless, let's just use `$this->is_woocommerce_active` directly below.

Reply:
Thanks @tofumatt , this has been refactored.